<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageResampleDlg</class>
 <widget class="QDialog" name="ImageResampleDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像重采样</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonInput">
        <property name="text">
         <string>输入</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditOuput"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOutput">
        <property name="text">
         <string>输出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonRasterIO">
        <property name="text">
         <string>RasterIO方法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonGDALWarp">
        <property name="text">
         <string>GDALWarp方法</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
